src/problems/hanoi.clj

```diff
@@ -0,0 +1,35 @@
+(ns problems.hanoi
+  (require [clojure.set :as set]))
+
+;;scardycat.clj was a practice file; this file is (hanoi.clj) our final submission
```

Comment:
You wouldn't want to "turn in" `scardycat.clj` (or include it in a pull request). Exploratory spikes like that are a good idea and make a lot of sense, but you don't want to leave them in the code base.

A common approach is to create a branch for the spike, and then switch over to a new branch (or `master` if you want to work directly there) when you're ready to write the "production" code. That way you still have a record of the spike without cluttering up the production code base. You can then eventually delete the spike branch down the road when it's clear that whatever was learned/captured there isn't likely to matter/help moving forward.